src/styles/utils/variables.scss

```
@@ -0,0 +1,8 @@
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+$gray-elements: #f3f3f3;
+$yellow-stars: #ffde6a;
+$white: #fff;
+$active-color: #00ACDC;
```

Why did you create the two variables with the same values?
```suggestion
$gray-elements: #f3f3f3;
$yellow-stars: #ffde6a;
$white: #fff;
$active-color: #00ACDC;
```